nox/_parametrize.py

```diff
@@ -81,7 +81,7 @@ def __eq__(self, other: object) -> bool:
 def _apply_param_specs(param_specs: List[Param], f: Any) -> Any:
     previous_param_specs = getattr(f, "parametrize", None)
     new_param_specs = update_param_specs(previous_param_specs, param_specs)
-    setattr(f, "parametrize", new_param_specs)
+    f.parametrize = new_param_specs
```

Comment:
These are not any safer, and make the reader think they are doing something different than the default syntax (which they do not). (Bugbear)

Reply:
Agreed 👍🏻